quantum/rgb_matrix.h

```diff
@@ -104,11 +104,13 @@ void rgb_matrix_indicators_user(void);
 void rgb_matrix_init(void);
 
 void        rgb_matrix_set_suspend_state(bool state);
+bool        rgb_matrix_get_suspend_state(void);
```

Comment:
These should really be `set_suspended()` and `is_suspended()`, but given `set_suspend_state()` already exists and is probably in use, that'll be a breaking change.

Reply:
I can cahnge the is/get function, since it's being added here.